### DISCUSSION TITLE
Sensor end alerts

### DISCUSSION THREAD

**Discussion by zibra** [j-kaltes/Juggluco#26] [Ideas]
It will be good to have few alerts reminding about sensor end. My suggestion will be to ring:
- 4h or 8h before the end
- 1h or 2h (or both) before the end
- when sensor ends

Alerts before the end will be useful to prepare, so you can f.e. start new sensor earlier to keep continous readings

**Comment by j-kaltes** [comment]
After starting a new sensor, Juggluco redirects the user to the calendar app in which the user can set the reminders they like.

**Reply by zibra** [reply]
True, but you need to set it for every sensor. Advantage of in-app alerts for this is that user can set it once and it will be applied for every sensor.
